bindings/nodejs/package.json

```diff
@@ -1,33 +1,47 @@
 {
   "name": "opendal",
   "author": "Databend Authors <opensource@datafuselabs.com>",
-  "license": "Apache-2.0",
   "version": "0.0.1",
-  "description": "",
   "main": "index.js",
-  "dependencies": {
-    "@node-rs/helper": "^1.2.1"
-  },
-  "devDependencies": {
-    "@napi-rs/cli": "^1.0.0"
-  },
+  "types": "index.d.ts",
+  "description": "",
   "napi": {
     "name": "opendal",
     "triples": {
-      "defaults": false,
       "additional": [
-        "x86_64-apple-darwin",
-        "x86_64-unknown-linux-gnu"
+        "aarch64-apple-darwin",
```

Comment:
**suggestion**: Since the develop work of nodejs binding is at very early stage. How about only keep a small set of targets? We can add freebsd, musl, android support in the future.

Reply:
SGTM.